### PR TITLE
feat(plasma-tokens): Saving themes artifacts from diez compile

### PR DESCRIPTION
**Pull request by neretin-trike**
Добавил результат компиляции тем инструментом `diez`  в директорию `legacyThemes`. Теперь это статичные токены, которые не будут каждый раз заново генерироваться.

Удалил внутренний пакет `design-language` из `plasma-tokens`.

:baby_chick: Download canary assets:

[colors--canary.489.4784360381.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.489.4784360381.xml)  
[color_sberHealth_ios-swift--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.489.4784360381.swift)  
[color_sberHealth_kotlin--canary.489.4784360381.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.489.4784360381.kt)  
[color_sberHealth_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.489.4784360381.ts)  
[color_sbermarket_business_ios-swift--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.489.4784360381.swift)  
[color_sbermarket_business_kotlin--canary.489.4784360381.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.489.4784360381.kt)  
[color_sbermarket_business_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.489.4784360381.ts)  
[color_sbermarket_ios-swift--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.489.4784360381.swift)  
[color_sbermarket_kotlin--canary.489.4784360381.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.489.4784360381.kt)  
[color_sbermarket_metro_ios-swift--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.489.4784360381.swift)  
[color_sbermarket_metro_kotlin--canary.489.4784360381.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.489.4784360381.kt)  
[color_sbermarket_metro_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.489.4784360381.ts)  
[color_sbermarket_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.489.4784360381.ts)  
[color_sbermarket_selgros_ios-swift--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.489.4784360381.swift)  
[color_sbermarket_selgros_kotlin--canary.489.4784360381.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.489.4784360381.kt)  
[color_sbermarket_selgros_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.489.4784360381.ts)  
[color_sbermarket_wlbusiness_ios-swift--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.489.4784360381.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.489.4784360381.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.489.4784360381.kt)  
[color_sbermarket_wlbusiness_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.489.4784360381.ts)  
[color_sberprime_ios-swift--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.489.4784360381.swift)  
[color_sberprime_kotlin--canary.489.4784360381.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.489.4784360381.kt)  
[color_sberprime_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.489.4784360381.ts)  
[PlasmaTokensColor--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.489.4784360381.swift)  
[shadow_sbermarket_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.489.4784360381.ts)  
[typo_mage_ios-swift--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.489.4784360381.swift)  
[typo_mage_kotlin--canary.489.4784360381.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.489.4784360381.kt)  
[typo_mage_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.489.4784360381.ts)  
[typo_plasma_ios-swift--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.489.4784360381.swift)  
[typo_plasma_kotlin--canary.489.4784360381.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.489.4784360381.kt)  
[typo_plasma_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.489.4784360381.ts)  
[typo_ruler_ios-swift--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.489.4784360381.swift)  
[typo_ruler_kotlin--canary.489.4784360381.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.489.4784360381.kt)  
[typo_ruler_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.489.4784360381.ts)  
[typo_sage_ios-swift--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.489.4784360381.swift)  
[typo_sage_kotlin--canary.489.4784360381.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.489.4784360381.kt)  
[typo_sage_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.489.4784360381.ts)  
[typo_sbermarket_ios-swift--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.489.4784360381.swift)  
[typo_sbermarket_kotlin--canary.489.4784360381.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.489.4784360381.kt)  
[typo_sbermarket_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.489.4784360381.ts)  
[typo_soulmate_ios-swift--canary.489.4784360381.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.489.4784360381.swift)  
[typo_soulmate_kotlin--canary.489.4784360381.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.489.4784360381.kt)  
[typo_soulmate_react-native--canary.489.4784360381.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.489.4784360381.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.204.0-canary.489.4784360381.0
  npm install @salutejs/plasma-core@1.111.0-canary.489.4784360381.0
  npm install @salutejs/plasma-hope@1.204.0-canary.489.4784360381.0
  npm install @salutejs/plasma-icons@1.140.0-canary.489.4784360381.0
  npm install @salutejs/plasma-temple@1.154.0-canary.489.4784360381.0
  npm install @salutejs/plasma-tokens@1.47.0-canary.489.4784360381.0
  npm install @salutejs/plasma-ui@1.186.0-canary.489.4784360381.0
  npm install @salutejs/plasma-web@1.204.0-canary.489.4784360381.0
  npm install @salutejs/plasma-cy-utils@0.54.0-canary.489.4784360381.0
  npm install @salutejs/plasma-sb-utils@0.109.0-canary.489.4784360381.0
  npm install @salutejs/plasma-tokens-android@2.54.0-canary.489.4784360381.0
  npm install @salutejs/plasma-tokens-ios-swift@2.54.0-canary.489.4784360381.0
  # or 
  yarn add @salutejs/plasma-b2c@1.204.0-canary.489.4784360381.0
  yarn add @salutejs/plasma-core@1.111.0-canary.489.4784360381.0
  yarn add @salutejs/plasma-hope@1.204.0-canary.489.4784360381.0
  yarn add @salutejs/plasma-icons@1.140.0-canary.489.4784360381.0
  yarn add @salutejs/plasma-temple@1.154.0-canary.489.4784360381.0
  yarn add @salutejs/plasma-tokens@1.47.0-canary.489.4784360381.0
  yarn add @salutejs/plasma-ui@1.186.0-canary.489.4784360381.0
  yarn add @salutejs/plasma-web@1.204.0-canary.489.4784360381.0
  yarn add @salutejs/plasma-cy-utils@0.54.0-canary.489.4784360381.0
  yarn add @salutejs/plasma-sb-utils@0.109.0-canary.489.4784360381.0
  yarn add @salutejs/plasma-tokens-android@2.54.0-canary.489.4784360381.0
  yarn add @salutejs/plasma-tokens-ios-swift@2.54.0-canary.489.4784360381.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-489/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-489/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-489/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-489/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-489/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-489/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-489/temple-storybook/

**Review by Yakutoc — APPROVED** [review]

File: packages/plasma-tokens/package.json [hunk 1]
```diff
@@ -48,6 +44,7 @@
         "styled-components": "^5.1.1"
     },
     "devDependencies": {
+        "@diez/web-sdk-common": "10.6.0",
```
Comment (Yeti-or) [inline packages/plasma-tokens/package.json:47]: не совсем понял зачем

File: packages/plasma-tokens/data/ds.ts [hunk 2]
```diff
@@ -17,11 +17,8 @@ import {
     OverlayTokens,
 } from '@salutejs/plasma-tokens-utils';
 
-import { DesignLanguage } from '../design-language/build/diez-plasma-tokens-web';
```
Comment (Yeti-or) [inline packages/plasma-tokens/data/ds.ts:20]: а тут только для plasma-web ? или я что-то запутался

 нужно тотже процесс делать для:

```
plasma-tokens-b2b/
plasma-tokens-b2c/
plasma-tokens-web/
```

??

File: packages/plasma-tokens/package.json (hunk 1 above)
Comment (neretin-trike) [reply packages/plasma-tokens/package.json:47]: эта зависимость импортится как `require` в файле index.js, который автоматически был сгенирирован диезом. Раньше они лежали в директрии design-language, где SDK был, но поскольку я её удалил, а файлы для тем остались, пришлось ставить доп зависимость, но она вроде как не большая, в package-lock'e тянет всего две сущности.

File: packages/plasma-tokens/data/ds.ts (hunk 2 above)
Comment (neretin-trike) [reply packages/plasma-tokens/data/ds.ts:20]: не, это актуально только для ui тем. Web в конце потому, что это режим компиляции, который задаётся при использовании диеза. 

На счёт остальных либ не уверен, что это надо делать, т.к. дальше мы их вроде не будем развивать, поскольку все новые схемы будут лежать в plasma-tokens

Comment (Yeti-or) [reply packages/plasma-tokens/data/ds.ts:20]: окей
2) заведи задачи на эти либы тоже пото сделаем

**Review by Yeti-or — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-489/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-489/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-489/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-489/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-489/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-489/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-489/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Pull request merged**